lib/widgets/recent_dm_conversations.dart

```diff
@@ -148,16 +148,31 @@ class RecentDmConversationsItem extends StatelessWidget {
             const SizedBox(width: 8),
             Expanded(child: Padding(
               padding: const EdgeInsets.symmetric(vertical: 4),
-              child: Text(
+              child: Text.rich(
+                TextSpan(
+                  children: [
+                    TextSpan(text: title),
+                    ...(switch (narrow.otherRecipientIds) {
+                      // self-DM
+                      [] => [UserStatusEmoji.asWidgetSpan(userId: store.selfUserId,
+                              fontSize: 17, textScaler: MediaQuery.textScalerOf(context))],
+                      // 1:1-DM
+                      [final otherUserId] =>
```

Comment:
Let's pull all this out of the return-expression and into a local variable, to reduce the amount of logic that's in the one expression.

In fact, looking at where `title` comes from: it's a switch on exactly the same three cases as this. So how about making `title` an InlineSpan that includes this?